airflow/providers/google/cloud/operators/automl.py

```diff
@@ -676,10 +676,25 @@ def execute(self, context: Context):
             )
 
 
+@deprecated(
+    reason=(
+        "Class `AutoMLTablesListColumnSpecsOperator` has been deprecated and will be removed after 31.12.2024. "
+        "AutoML platform no longer supports tabular datasets after its integration with Cloud Translation and Vertex AI. "
+        "Please refer to https://cloud.google.com/vertex-ai/docs/start/migrating-to-vertex-ai and "
+        "https://cloud.google.com/translate/docs/advanced/automl-upgrade for more info about available alternatives."
+    ),
+    category=AirflowProviderDeprecationWarning,
+    action="error",
```

Comment:
```suggestion
```

If users wanted they could configure it by Python builtin [Warnings Control](https://docs.python.org/3/library/warnings.html#the-warnings-filter) we should not force it to the to the `error`.

Reply:
Removed the decorator.